src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml

```diff
@@ -139,6 +139,7 @@
     <class-descriptor class="edu.cornell.kfs.sys.businessobject.UserProcurementProfile" table="CU_USER_PROCUREMENT_PROFILE_T">
     	<field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
     	<field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true"/>
+    	<field-descriptor name="active" column="ROW_ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion"/>
```

Comment:
I mentioned this in the sql review as well, but I think this should probably be just ACTV_IND (see: https://kualico.atlassian.net/wiki/x/Q4AZ).